src/common/utils.ts

```diff
@@ -121,8 +121,20 @@ export class ByteArrayMap<T> {
   }
 }
 
-function filePathRelativeTo(base: string, filePath: string): string {
-  return new URL(filePath, new URL(base, 'file://')).pathname;
+export function filePathRelativeTo(base: string, filePath: string): string {
+  // Normalize the base path to convert any Windows backslashes to forward slashes
+  // This is necessary because the URL object expects forward slashes
+  const normalizedBase = base.replace(/\\/g, '/');
+
+  // Create a URL object with the file protocol and the normalized base path
+  const baseURL = new URL(normalizedBase, 'file:///');
```

Comment:
This comment suggests an alternative way of creating the URL that might be more windows friendly: https://github.com/nodejs/node/issues/20080#issuecomment-381686455


```javascript
new URL(`file:///${path}`)
```

If that works, it'd be nice to avoid manually adjusting the file separator